README.md

```diff
@@ -1 +1,21 @@
-# esb-connector-stackexchange
\ No newline at end of file
+# esb-connector-stackexchange
+
+The StackExchange Connector allows you to access the [StackExchange REST API](https://api.stackexchange.com/docs) 
+through WSO2 Enterprise Integrator (WSO2 EI). StackExchange is a network of question-and-answer websites on topics in 
+diverse fields, each site covering a specific topic, where questions, answers, and users are subject to a reputation 
+award process. The reputation system allows the sites to be self-moderating.
+
+## Getting started
+
```

Comment:
Add "Download and install the connector" and "Configuring the connector operations" sections

https://github.com/wso2-extensions/esb-connector-googlespreadsheet#download-and-install-the-connector

Reply:
Fix with dd183b6